src/main/java/race/Utils/InputUtils.java

```diff
@@ -0,0 +1,29 @@
+package race.Utils;
+
+import java.util.Scanner;
+
+public class InputUtils {
+    public static final String INPUT_NUMBER_OF_CAR = "자동차 대수는 몇 대 인가요?";
+    public static final String INPUT_NUMBER_OF_MOVEMENT = "시도할 회수는 몇 회 인가요?";
```

Comment:
```suggestion
    private static final String INPUT_NUMBER_OF_CAR = "자동차 대수는 몇 대 인가요?";
    private static final String INPUT_NUMBER_OF_MOVEMENT = "시도할 회수는 몇 회 인가요?";
```
외부 호출이 없는 상수의 접근제어는 private으로 막는게 좋을 것 같네요